### PR TITLE
Revised README

### PR DESCRIPTION
**Pull request by jennyengelman**
Changed PongMates to Need One
Updated and Elaborated Architecture section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -27,4 +30,4 @@ PongMates is designed to improve social integration at Dartmouth through the gam
 
 ## Acknowledgments
 
-We'd like to thank our faculty advisor Lorie Loeb and mentor Ijemma Onwuzulike (Dartmouth class of ‘19) for their guidance and the hard work they've contributed to helping us succeed in achieving our goals within this project, as well as outside.
+We'd like to thank our faculty advisor Lorie Loeb and mentor Ijemma Onwuzulike (Dartmouth class of ‘19) for their guidance and the hard work they've contributed to helping us succeed in achieving our goals within--as well as outside--this project.
```
Comment (ijemmao) [inline README.md]: I would change the double dash (--) to just commas

File: README.md [hunk 2]
```diff
@@ -1,13 +1,16 @@
...
+Need One is designed to improve social integration at Dartmouth through the game of pong. Need One will serve as a platform to connect a user with a partner to play with, either by searching for or posting a game. We hope that this will become an easy way for new members of the Dartmouth community, such as freshmen or transfer students, to branch out and interact with new people. Additionally, the app could serve as a mechanism for upperclassmen to reach out to underclassmen and make Dartmouth a more inclusive place.
 
 ## Architecture
 
 * [Expo](https://expo.io/)
+- Expo is a tool for React Native applications that, by providing a prototype formate, aided our development process.
```
Comment (ijemmao) [inline README.md]: formate to format

**Review by ijemmao — CHANGES_REQUESTED** [review]
Solid README

**Comment by jennyengelman** [comment]
Were these the formatting changes you wanted?

**Pull request merged**